### ISSUE TITLE
ArgumentException thrown from KPM restore that I can't work around

### ISSUE THREAD

**Issue by MattGal** [aspnet/DNX#1396]
(Apologies if this is a dupe... I just typed this whole issue up, found more details, then can't find it again.)

I have hit an issue with the specific dependencies of my project that somehow survives both deleting the entire package cache as well as deleting every project.lock.json file.  I can help members of the DNX team to see it on my local machine, and have also stored off the proj folder and package cache from such a setup.

Interestingly, after deleting my package cache, I get the same error with a different package in 2 different projects in the same solution.  Regardless, it's easy to reproduce and shows a call stack the user should never see (as mentioned in #452)

Writing lock file C:\GIT\<Project path>\project.lock.json

System.ArgumentException: Can not add property Microsoft.AspNet.Diagnostics.Interfaces to Newtonsoft.Json.Linq.JObject. Property with the same name already exists on object.
   at Newtonsoft.Json.Linq.JObject.ValidateToken(JToken o, JToken existing)
   at Newtonsoft.Json.Linq.JContainer.InsertItem(Int32 index, JToken item, Boolean skipParentCheck)
   at Newtonsoft.Json.Linq.JObject.InsertItem(Int32 index, JToken item, Boolean skipParentCheck)
   at Newtonsoft.Json.Linq.JContainer.AddInternal(Int32 index, Object content, Boolean skipParentCheck)
   at Newtonsoft.Json.Linq.JContainer.Add(Object content)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WriteObject[TItem](IEnumerable`1 items, Func`2 writeItem)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WritePackageDependencySet(PackageDependencySet item)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WriteObject[TItem](IEnumerable`1 items, Func`2 writeItem)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WriteLibrary(LockFileLibrary library)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WriteObject[TItem](IEnumerable`1 items, Func`2 writeItem)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.WriteLockFile(LockFile lockFile)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.Write(Stream stream, LockFile lockFile)
   at Microsoft.Framework.Runtime.DependencyManagement.LockFileFormat.Write(String filePath, LockFile lockFile)
   at Microsoft.Framework.PackageManager.RestoreCommand.WriteLockFile(String projectLockFilePath, List`1 graphItems, PackageRepository repository)
   at Microsoft.Framework.PackageManager.RestoreCommand.<RestoreForProject>d__79.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Framework.PackageManager.RestoreCommand.<>c__DisplayClass77_0.<<ExecuteCommand>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at Microsoft.Framework.PackageManager.RestoreCommand.<>c__DisplayClass89_0`1.<<ForEachAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Framework.PackageManager.RestoreCommand.<ExecuteCommand>d__77.MoveNext()

Restore failed
Can not add property Microsoft.AspNet.Diagnostics.Interfaces to Newtonsoft.Json.Linq.JObject. Property with the same name already exists on object.

**Comment by davidfowl** [comment]
I'm sure I just closed this bug and pointed you to the issue on home so I'll do that here and write the short version.

https://github.com/aspnet/Home/issues/368#issuecomment-78107172

TL;DR 

We renamed the TFMS and kre is now broken against the latest packages. The fix is to update everything. kvm -> dnvm. k -> dnx
